src/nav/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import "./index.css";
+
+export default class Nav extends React.Component {
```

Comment:
why have you decided to write it as class component?
change it to functional component please.